### DISCUSSION TITLE
Is anyone else seeing duplicate commits when using the Merge Queue?

### DISCUSSION THREAD

**Discussion by jeremypress** [community/community#36568] [Pull Requests]
We've enabled the merge queue beta, and we're seeing that occasionally gh will merge 2 commits with the same title for 1 pr. These will have 2 different sha hashes, but 1 will be empty and 1 will contain the changes. Anyone else running into this?

**Comment by ossanna16** [comment]
Hey there @jeremypress and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by andyscott** [comment]
We're seeing the same exact behavior where the merge queue will sometimes merge a single commit into the main branch twice. The second "copy" will be an empty commit with the same commit message as the first.

**Comment by promiseofcake** [comment]
We are seeing similar issues as well.

**Comment by manon-traverse** [comment]
This is a daily occurrence for us as well.

**Comment by franzliedke** [comment]
Same here. 😕

**Comment by LaurisJakobsons** [comment]
We are having the same issue as well.

**Comment by Gabriella439** [comment]
We are seeing the same issue, too.  It doesn't happen on all commits merged, but some of them

**Comment by froeba** [comment]
We've started to see this issue today as well.  It can mess up our CI as the empty commit is run instead of the commit with data, causing our scripts in the monorepo to skip publishing packages because it doesn't detect changes.

**Comment by santoshD93** [comment]
Same here!

**Comment by teoljungberg** [comment]
Ditto. Noticed due to duplicated email notifications (one merge notification from the merge-queue and one merge notifications from the author's account).
